iquip/apps/scheduler.py

```diff
@@ -234,6 +302,35 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         """Extended."""
         super().__init__(name, parent=parent)
         self.schedulerFrame = SchedulerFrame()
+        self.schedulerFrame.queueView.fetched_event.connect(self.displayMenu)
+
+
+    @pyqtSlot(QMouseEvent)
+    def displayMenu(self, event: QMouseEvent):
+        """Displays the menu pop-up.
+
+        Args:
+            event: Information holder containing the clicked position.
+        """
+        for i in range(self.schedulerFrame.model.rowCount()):
+            index = self.schedulerFrame.model.index(i)
+            if self.schedulerFrame.queueView.rectForIndex(index).contains(event.pos()):
+                rid = self.schedulerFrame.model.data(index).arginfo["rid"]
+                menu = QMenu(self.schedulerFrame)
+                edit = menu.addAction("Edit")
+                delete = menu.addAction("Delete")
+                request_termination = menu.addAction("Request termination")
+                # TODO(giwon2004) Remove icon space from the menu list (use menu.setStyleSheet)
+
+                action = menu.exec_(self.schedulerFrame.mapToGlobal(QPoint(0,25)) + event.pos())
```

Comment:
If I calculate the position in the `SchedulerFrame`, then it will be just right at the position `event.pos()`.

However when it is in the `SchedulerApp`, the header (image below) takes 25 of space:
![image](https://github.com/snu-quiqcl/iquip/assets/66408173/ddd60ed1-4edb-4258-9381-8d81be0e53a5)
and this `QPoint(0, 25)` is the adjustment of it.

Reply:
However, the value of the margin changes when this is opened by a separate window, so I need to find a way to solve this.

Reply:
Okay, thank you for the explanation.
Then how about using `event.globalPos()` as shown in the _common usage_ example of [`QMenu.exec()`](https://doc.qt.io/qt-5/qmenu.html#exec-1)?

Reply:
Oh wow it worked, I did not know that existed. Thank you!